### PR TITLE
Created the pr_template

### PR DESCRIPTION
**Pull request by gyauelvis**
#### THIS IS A PULL REQUEST TEMPLATE! 

⚠️ This helps you complete a pull request. Please follow the instructions below. Reminder that you are bound to abide by the [Code of Conduct](/CODE_OF_CONDUCT.md) ⚠️ 

### Submission Checklist

---

Please complete the below checklist. Failure to follow these instructions may result in your pull request being closed without review.

### What issue does this PR closes?
[pr template issue](https://github.com/JhohannesK/TipoHiZ/issues/156)

### Description of file changes
- Created `pull_request_template` in the `.github` repository

### Is this PR still in progress or completed?
- [ ] This PR is still a work in progress (WIP)
- [x] This PR is completed and ready for review

### Screenshot if applicable
image

### Additional Information

- [ ] Did you edit the package.json or pnpm.yaml? If yes, explain reason for modification and add the `dependencies` label to this PR. <br>

### CODE REVIEW

File: pnpm-lock.yaml
Comment (JhohannesK) [inline pnpm-lock.yaml:1]: How come you have changes in your lock file?
Exclude this.

File: .github/pull_request_template.md
Comment (JhohannesK) [inline .github/pull_request_template.md:1]: I added some changes this file in your previous PR. Please check and add them to this one.

**Review by JhohannesK — CHANGES_REQUESTED** [review]
Since you've not added anything to the package.json, there's no reason to push a lock file.
Please omit it.

**Comment by gyauelvis** [comment]
> Since you've not added anything to the package.json, there's no reason to push a lock file. Please omit it.

Sure. Will Omit it

File: .github/pull_request_template.md
Comment (gyauelvis) [reply .github/pull_request_template.md:1]: You changed the word address to close--added
and then added a new checklist. I have added all of them

**Pull request closed without merging**